pkg/server/status.go

```diff
@@ -608,6 +608,7 @@ func (s *statusServer) Details(
 	if err != nil {
 		return nil, grpcstatus.Errorf(codes.InvalidArgument, err.Error())
 	}
+	telemetry.Count("web-ui.health")
```

Comment:
The name of this feature is misleading. This service method is used for a bunch of things, not just the web UI. As-is, you're going to get over-reporting.

If you want to specialize to web UI access you'll need to detect that the client is the web UI, presumably via a field override in DetailsRequest.

If you're intending to count all accesses to the RPC endpoint, you'll need a better feature name.

Reply:
It's intended to be about monitoring in general, so I've updated it to be `monitoring.*` instead of `web-ui.*`. Good point!